### DISCUSSION TITLE
How to set the explicit boundaries of a histogram being sent to the collector?

### DISCUSSION THREAD

**Discussion by keep94** [open-telemetry/opentelemetry-go#2513] [Q&A]
I see two histogram like types in the code.

1. metric.Float64Histogram - This type can send histograms to the collector, but the type provides no obvious way to set the explicit boundaries of the histogram
2. histogram.Aggregator - This type has an obvious way to set the explicit boundaries, but no obvious way to send the histogram to the collector.

Do I somehow need to use both of these types to send histograms to the collector?

**Comment by MrAlias** [comment]
You can configure explicit histogram boundaries with the [`WithExplicitBoundaries`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/metric@v0.26.0/aggregator/histogram#WithExplicitBoundaries) option when creating a histogram `Aggregator`. That aggregator can be associated with your `Controller` via a `Processor` (e.g. [this](https://github.com/open-telemetry/opentelemetry-go/blob/c5776cc953121df58f59d8870caab7e0afbae4bc/example/prometheus/main.go#L45-L51)). You can also associate whatever exporter you want to use with that `Controller` by passing it with the [`WithExporter`](https://pkg.go.dev/go.opentelemetry.io/otel/sdk/metric@v0.26.0/controller/basic#WithExporter) option. For an `otlpmetric` exporter using gRPC to communicate with the collector you can something similar to [this](https://github.com/open-telemetry/opentelemetry-go/blob/c5776cc953121df58f59d8870caab7e0afbae4bc/exporters/otlp/otlpmetric/otlpmetricgrpc/example_test.go#L54).
